### PR TITLE
Add new tips on startup, fix some outdated

### PR DESCRIPTION
**Pull request by DelazJ**
and emphasize some items with bold

### CODE REVIEW

File: src/app/qgstipfactory.cpp [hunk 1]
```diff
@@ -97,19 +97,18 @@ QgsTipFactory::QgsTipFactory() : QObject()
                         " move them around with the arrow keys. For accurate positioning use the"
                         " <strong>Position and Size</strong> section, which can be found in the"
                         " tab <strong>Item Properties -> Position and Size</strong>."
-                        " The other move tool (map canvas icon with right arrow) allows one"
-                        " to move the map content within a map frame."
+                        " The other move tool (a scrolled map icon with four oriented arrows)"
+                        " allows one to move the map content within a map frame."
```
Comment (m-kuhn) [inline src/app/qgstipfactory.cpp]: Is *allows one* a proper English expression? I have a feeling that this is a literal translation from German. CC @NathanW2 @nyalldawson

Comment (nyalldawson) [inline src/app/qgstipfactory.cpp]: It's fine, just a little formal/"posh" sounding.

Comment (m-kuhn) [inline src/app/qgstipfactory.cpp]: Or "allows moving"?

Comment (m-kuhn) [inline src/app/qgstipfactory.cpp]: > Better would be "allows you to...."

That makes sense, in German the "one" construct is often used to circumvent the question if the user should be addressed with a formal you "Sie" or an informal you "Du".

**Comment by m-kuhn** [comment]
More ideas

 * Mention `F7`, the ultimate "open styling dock" key
 * Usage of images because plain text on application start always triggers my brain stem's *fight and flight* mode.

**Comment by DelazJ** [comment]
> Mention F7, the ultimate "open styling dock" key

As a new tip? Sounds good. Btw, when you right)-click on a panel or toolbar to show their list, would also help to add behind each panel or toolbar name, any corresponding shortcut. Only Processing toolbox is currently provided.
> Usage of images because plain text on application start always triggers my brain stem's fight and flight mode

Not sure to understand. You mean add more images in the current tips or replace tips by more images? What kind of images are you looking for? Any example?

**Comment by m-kuhn** [comment]
E.g.

> " The other move tool (a scrolled map icon with four oriented arrows)"

Replace it with the real icon.

Or just show off some cool symbols and effects. I'm sure @nirvn has some nice ideas.

**Comment by DelazJ** [comment]
@m-kuhn
* I tried to add the icons but note sure if it's the way it should be set (using  tag with svg file - i thought it was good to directly use the source file instead of generating a png icon that could be obsolete one day.)
* Insert the F7 shortcut in an existing tip instead of creating new one
* Add a new tip on the use of shortcuts

About cool symbols or aesthetics, allow me to leave the place for more inspired people :)

**Comment by m-kuhn** [comment]
Sounds good to me.
There is a quotation problem, see here:
https://dash.orfeo-toolbox.org/viewBuildError.php?type=1&buildid=253364

**Comment by DelazJ** [comment]
Fixed and all is green

**Pull request merged**